modules/effects/migrations/migration.json

```
@@ -20,6 +20,11 @@
       "description": "The road to v15 beta",
       "version": "15-beta",
       "factory": "./15_0_0-beta/index"
+    },
+    "ngrx-effects-migration-18": {
+      "description": "use ngrx operators",
+      "version": "18",
+      "factory": "./18_0_0/index"
```

This can be included in the beta release. 
(The `18_0_0_` directory should also be renamed with this change)

```suggestion
    "ngrx-effects-migration-18-beta": {
      "description": "use ngrx operators",
      "version": "18-beta",
      "factory": "./18_0_0-beta/index"
```